README.md

```diff
@@ -17,21 +17,29 @@ dotnet --version
 ``` 
 to determine which version you have installed, or visit the [.NET docs][11] for more information.
 
-## [Description:][1]
+## [Project Description:][1]
 
-### Functional Requirements
+- This project started off with a simple Echo Server, and has been built out into an HTTP Server.
+- This HTTP server includes routes, requests, and responses.
+- The routes must be customizable with a URL, a verb, and an action to take when the route is called.
 
-A user should be able to interact with the echo server as follows:
+## Functional Requirements
 
-- When a client sends a message to the server, the server sends a response back to the client containing the original message.
-- A client can send multiple messages to the server and get the echoed response back each time.
-- Multiple clients can send messages to server and get back their proper responses.
+A user should be able to interact with the HTTP server as follows:
 
-### Implementation Requirements
+* When a client sends a properly formatted request to the server, the server sends an appropriate response back to the client.
+* A client can send different HTTP requests to the server and get the appropriate response back each time.
+* Different clients can send messages to server and get back their proper responses.
+* The server should be able to handle 200, 300, and 400-level responses.
 
-- The server should establish a socket connection with the client using a low-level socket library. The goal of this exercise is to work with sockets directly.
-- The server should accept and return streams of data rather than raw strings.
-- The echo server should be covered by a robust suite of tests.
+## Implementation Requirements
+
+* The server should establish a socket connection with the client using a low-level socket library.
+* The server should accept and return streams of data rather than raw strings.
+* The HTTP server should be covered by a robust suite of unit tests.
+* The HTTP server should pass all of the tests covered in `01_getting_started` in the [HTTP Server Spec](https://github.com/8thlight/http_server_spec).
```

Comment:
Pretty sure you can just link to the code in your repo.

```suggestion
* The HTTP server should pass all of the tests covered in `01_getting_started` in the [HTTP Server Spec](http_server_spec/features/01_getting_started).
```

Reply:
This is really cool - I was trying to figure out how to do this/if it was even possible so yay, learned something new and fun!